examples/src/main/scala/examples/commons/SimpleBoxTransaction.scala

```diff
@@ -61,7 +61,7 @@ case class SimpleBoxTransaction(from: IndexedSeq[(PublicKey25519Proposition, Non
 
   override lazy val json: Json = Map(
     "id" -> Base58.encode(id).asJson,
-    "newBoxes" -> newBoxes.map(b => Base58.encode(b.id).asJson).asJson,
+    "newBoxes" -> newBoxes.map(b => Base58.encode(b.id).asJson).toSeq.asJson,
```

Comment:
Why is this change needed?

Reply:
I don't know why, but after updating circe's version this started to produce compile error.